drivers/jason0x43-wemo_dimmer.groovy

```diff
@@ -61,8 +61,31 @@ def off() {
 }
 
 def setLevel(value) {
-    log.info("Setting level to $value")
-    def binaryState = value > 0 ? 1 : 0;
+    def binaryState = 1
+    
+    if (value > 0 && value <= 100) {
+        binaryState = 1
+    } else if (value == 0) {
+        binaryState = 0
+    } else {
+        binaryState = 1
+        value = 100
+    }
+    log.info("SL1: Setting level to $value with state to $binaryState")
+    parent.childSetBinaryState(device, binaryState, value)
+}
+
+def setLevel(value, binaryState) {
```

Comment:
Isn't the second argument in this overload the duration? (https://docs.hubitat.com/index.php?title=Driver_Capability_List#SwitchLevel)